astore/client/commands/delete.go

```diff
@@ -0,0 +1,38 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type DeleteCommand struct {
+	*cobra.Command
+	root *Root
+
+	name string
+}
+
+func (uc *Delete) Run(cmd *cobra.Command, args []string) error {
+	//TODO
+	return nil
+}
+
+type Delete struct {
+	*cobra.Command
+	root *Root
+	isDryRun bool
```

Comment:
Did not indicate in the ticket, and can be in a follow up: but when a sid is specified, this may result in multiple uids removed [1]. The tool should  default to ask for confirmation, there should be a flag to disable it.


[1]: eg, sid identifies an artifact, like the "tools/enkit" binary. uid identifies a specific version, like the enkit we released 2 months ago, so if you remove a sid, the intent is to remove all versions of enkit from the repo, while if you remove the uid, you are removing a "single file".